docs/install/windows_setup.md

```diff
@@ -33,9 +34,19 @@ To build and install MXNet yourself, you need the following dependencies. Instal
 After you have installed all of the required dependencies, build the MXNet source code:
 
 1. Download the MXNet source code from [GitHub](https://github.com/dmlc/mxnet).
-2. Use [CMake](https://cmake.org/) to create a Visual Studio solution in ```./build```.
-3. In Visual Studio, open the solution file,```.sln```, and compile it.
-These commands produce a library called ```mxnet.dll``` in the ```./build/Release/``` or ```./build/Debug``` folder.
+2. Open native tools command prompt for Visual Studio 2015 ( choose x64 or x86 native as required)
```

Comment:
There's no need run cmake gui in tools command prompt.

Reply:
Not doing so results in errors from CUI being unable to discover VS tools.

Reply:
no.If there's no problem in your system.gui cmake can find vs with direct run.